src/pages/networks/forms/NetworkDefaultACLReadOnly.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FormikProps } from "formik";
+import type { FC } from "react";
+import type { NetworkFormValues } from "./NetworkForm";
+import { Icon } from "@canonical/react-components";
+import { conjugateACLAction } from "util/helpers";
+
+const NetworkDefaultACLReadOnly: FC<{
+  formik: FormikProps<NetworkFormValues>;
```

Comment:
Can we reuse the same `values?: Record<Direction, string>` interface as for the edit mode? Because we will want to reuse this on the instance nic devices and there we'll have a different formik type. The interface for edit mode will be possible to adapt to instance and profile nic devices.

Reply:
The idea was not to receive the formik object, but to unpack the values in the caller already. Reusing the interface for values as a record as you defined it in edit mode.

This will unlock reusing this component in instance and profile network config pages that have a different formik type.